tests/MongoDB.Driver.Tests/AuthenticationTests.cs

```diff
@@ -347,6 +352,16 @@ private void AssertAuthenticationSucceeds(MongoClientSettings settings, bool asy
                 {
                     client.ListDatabaseNames().ToEnumerable().ToList();
                 }
+                if (shouldSpeculativelyAuthenticateIfPossible &&
+                    CoreTestConfiguration.ServerVersion >= Feature.SpeculativeAuthentication.FirstSupportedVersion)
```

Comment:
Should it be `Feature.SpeculativeAuthentication.IsSupported`?

Reply:
Done